### DISCUSSION TITLE
Creo que me mandé una macana

### DISCUSSION THREAD

**Discussion by LaSudestada** [dds-jv/foro#410] [Consultas generales]
Buenas tardes profesor
Soy Ulises Suhr, alumno suyo de los viernes a la noche
Recién estaba mirando los links que agregó en el campus y entré en el TPI1 Macowins
Eso me llevó a github classroom y me salió una lista de números, que yo creía que era una acordeón o un desplegable y al hacerle clic al primero de la lista como que se me asignó en una aula, o algo por el estilo
Ahora me sale esto: "Once you accept this assignment, you will be granted access to the tpi1-macowins-2023-LaSudestada repository in the [dds-utn](https://github.com/dds-utn) organization on GitHub."
Puedo "Accept this assignment", pero por miedo a seguir haciendo macanas mejor no toco nada
Muchas gracias por su tiempo y disculpe las molestias!! Buen finde!!

**Comment by flbulgarelli** [comment]
¡Hola!

Uno de los enlaces había quedado apuntando al curso de 2023. Si volvés a ingresar al Campus Web deberías encontrar los enlaces de matriculación correctos a cada uno de los TPI. Si no encontraras tu email en la lista de registracion, avisanos. 

¡Saludos!
